src/pages/auth/ForgotPassword.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Trans, t } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+
+import Button from 'components/button/Button';
+import TextField from 'components/textfield/TextField';
+import { Panel, PanelContent } from 'components/panel/Panel';
+import { SignUp, TitleMessage } from './Utilities';
+import { ReactComponent as HandImage } from '../home/assets/images/forgot-password.svg';
```

Comment:
Please move `forgot-password.svg` to `./assets/images/forgot-password.svg`. Each module should have their own images.

I'm also thinking on creating a common assets folder for the logo and other common images. But the idea is that each module should have their own unique assets (if any)

Reply:
I agree that we need a common